Source/Plugins/CrossPlatformPluginGenerator/Assets/Microsoft.SpatialAudio.Spatializer.Unity/Plugins/Android/libs/arm64-v8a/libAudioPluginMicrosoftSpatializerCrossPlatform.so.meta

```diff
@@ -71,7 +71,7 @@ PluginImporter:
     second:
       enabled: 0
       settings:
-        CPU: X86
+        CPU: ARM
```

Comment:
Shouldn't this one be ARM64?

Reply:
Oh, wait. This change was to WSA, not to Android. OK, I get it.

Reply:
Gah, yep, I triple-checked but somehow this got messed up, fixing it.

Reply:
Ah yeah, whew, not an issue.